### PR TITLE
Sets homework

### PR DESCRIPTION
**Pull request by NarekM**

### CODE REVIEW

File: src/com/epam/homework/sets/Node.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.epam.homework.sets;
+
+public class Node<T extends Comparable<T>> {
```
Comment (naira-tamrazyan) [inline src/com/epam/homework/sets/Node.java]: Node can be a nested class inside your Set

**Review by naira-tamrazyan — APPROVED** [review]
Well done, Narek jan

File: src/com/epam/homework/sets/Node.java (hunk 1 above)
Comment (NarekM) [reply src/com/epam/homework/sets/Node.java]: Should I change it and make a nested class?

Comment (naira-tamrazyan) [reply src/com/epam/homework/sets/Node.java]: You may change it

Comment (NarekM) [reply src/com/epam/homework/sets/Node.java]: I changed Node to inner class and pushed

**Comment by naira-tamrazyan** [comment]
It.is ok, you may merge the code.

**Pull request merged**